### DISCUSSION TITLE
Cannot execute get-modflow after installing flopy

### DISCUSSION THREAD

**Discussion by qinghe-gm** [modflowpy/flopy#1651] [Q&A] [answered]
Hi all, 

I'm trying to install the executable modflow program (mf6) using Flopy. So I first installed python3, numpy, and Matplotlib, then I installed Flopy package. But when I tried to run get-modflow: Flopy, a bug occurred "-bash get-modflow command not found". Does anyone know how to fix this problem?

Any tip will be appreciated!

Thanks,
Qing

**Comment by wpbonelli** [comment, marked as answer]
Hi @Qing-sudo, `get-modflow` is currently an unreleased feature, although it should come out with `3.3.6` very shortly. Try removing your installed version of FloPy then install from the develop branch, e.g.

```
pip install pip install git+https://github.com/modflowpy/flopy.git
```

Since `develop` is the default branch the README reflects the development state of FloPy, not the official release. It is a bit confusing that the develop version shows by default since the README may be many people's entry point to FloPy, maybe we could clarify somewhere.
